io.github.tx2z.XtoMarkdown.yml

```
@@ -0,0 +1,242 @@
+# Flathub manifest for XtoMarkdown
+# This manifest is for Flathub submission - NO network access during build
+# For local development builds, use io.github.tx2z.XtoMarkdown.yml instead
+#
+# To submit to Flathub:
+# 1. Fork https://github.com/flathub/flathub
+# 2. Copy this file as io.github.tx2z.XtoMarkdown.yml
+# 3. Submit a pull request
+
+app-id: io.github.tx2z.XtoMarkdown
+runtime: org.kde.Platform
+runtime-version: "6.8"
```

```suggestion
runtime-version: "6.10"
```
Not the newest runtime